packages/toolpad-core/src/SignInPage/SignInPage.tsx

```diff
@@ -156,6 +158,11 @@ export interface SignInPageSlots {
    * @default Link
    */
   signUpLink?: React.JSXElementConstructor<LinkProps>;
+  /**
+   * Custom information added to the top header beneath the title
+   * @default Typography
+   */
+  headerInfo?: React.ElementType;
```

Comment:
By header you mean the title, right? But we've been calling "header" to something more like the AppBar on top of pages. So maybe instead of "header" it could be "heading" or "title"?

Reply:
It's the section that could contain a sub-heading, given the heading of the Sign In Page is "Sign In", but I get your concern about the conflict with the "header" being the app bar. We could call it `secondaryTitle`?

Reply:
That sounds ok, on second thought maybe "headerInfo" isn't bad either but it could get confusing if we end up adding a header with the theme switcher I guess...
Or even something like "subHeading" could work probably. Feel free to go with what you think is best in the end, just thought I'd mention this.

Reply:
Yes, on second thought I could probably add two slots called `title` and `subtitle` to be able to override that entire section. What do you think?

Reply:
`title` and `subtitle` sound super clear to me.